tools/integration_tests/run_tests_mounted_directory.sh

```diff
@@ -85,9 +85,11 @@ GODEBUG=asyncpreemptoff=1 go test ./tools/integration_tests/operations/...  -p 1
 sudo umount $MOUNT_DIR
 
 # Run tests with config "file-cache: max-size-in-mb" static mounting.
-echo "file-cache:
+echo "write:
```

Comment:
@ashmeenkaur @sethiay this echo command was the only source of conflict in this merge. Please confirm that this merge will work.

Reply:
@sethiay should we run these separately for different configs or is it fine to run them together?

Reply:
Ideally, we should run them separately.

Reply:
Done.